conf.json

```diff
@@ -1,18 +1,19 @@
 {
   "base": {
     "password": "AES256Key-32Characters1234567890",
-    "crypto": "aes-128-gcm"
+    "crypto": "aes-256-gcm",
```

Comment:
Change this file unless there is new config settings.
The old config settings should not be  changed.

Reply:
https://github.com/xiaoxiaoking/medusa/commit/de1a3fdfdb9877d2c8de4b6c7e14628db240b8ec